dags/multipod/maxtext_end_to_end.py

```diff
@@ -131,6 +131,23 @@
               "time_out_in_min": 60,
           },
       ],
+      "llama2-70b": [
+          {
+              "script_name": "tpu/llama2/70b/1_test_llama2_70b",
+              "cpu_device_type": CpuVersion.M1_MEGAMEM,
+              "cpu_zone": Zone.US_CENTRAL1_B.value,
+              "cluster_name": ClusterName.CPU_M1_MEGAMEM_96.value,
+              "time_out_in_min": 360,
```

Comment:
How big is the CPU cluster? With a 6-hour timeout, we could starve the other conversion tests depending on which executes first (the current XPK queuing timeout is [5 hours](https://github.com/GoogleCloudPlatform/ml-auto-solutions/blob/0c23cbb3bef31986a3ae523371f0d1cfe9bb8e0b/xlml/apis/task.py#L320-L322)).

Reply:
There is one host in this CPU cluster for the `m1_megamem_96`, @RissyRan  and I were discussing we could increase it more if need be. We have more quota although it is currently at 1 host since these `m1-megamem-96` 
 are somewhat expensive